float32 pressure	# static pressure measurement in millibar
float32 altitude	# ISA pressure altitude in meters
float32 temperature	# static temperature measurement in deg C
uint64 error_count
uint32 device_id	# Sensor ID that must be unique for each baro sensor and must not change
